src/components/earn/UnstakingModal.tsx

```diff
@@ -60,58 +64,78 @@ export default function UnstakingModal({ isOpen, onDismiss, stakingInfo }: Staki
 
   let error: string | undefined
   if (!account) {
-    error = 'Connect Wallet'
+    error = t({ id: 'wallet.connect', message: 'Connect a wallet' })
```

Comment:
Not sure if the text change here is intentional

Reply:
intentional as well for consistency across the app 👍